packages/cli/src/commands/list.js

```diff
@@ -5,7 +5,27 @@ import { makeRefIterator } from '@endo/daemon';
 import { withEndoHost } from '../context.js';
 import { parsePetNamePath } from '../pet-name.js';
 
-export const list = async ({ directory, follow, json }) =>
+const prettyValue = val => {
+  let result;
+  const type = typeof val;
+  if (type === 'string') {
+    result = `'${val}'`;
```

Comment:
I like using JSON.stringify in cases like this since it handles all the edge cases in a way that can be copy+pasted.

Reply:
What I really want is Node's `util.inspect`, which handles a bunch of stuff that chokes `JSON.stringify`.  You get that for free if you pass a naked value to `console.log`, but if you want to handle the string itself you need to be running in Node, which I don't think is an invariant we want here.  However, it occurs to me that since the value is the last thing on the line we can handle cases we don't know how to format but literally just passing the unformatted value as a second arg to `console.log`.

Reply:
Very good.  That simplifies quite a few things.